test/test_torch.py

```diff
@@ -6494,6 +6494,21 @@ def test_serialization_filelike_uses_readinto(self):
         b = torch.load(data)
         self.assertTrue(data.was_called('readinto'))
 
+    def test_load_error_msg(self):
+        expected_err_msg = (".*You can only torch.load from a file that is seekable. " +
+                            "Please pre-load the data into a buffer like io.BytesIO and " +
+                            "try to load from it instead.")
+        import sys
+        if sys.version_info < (3,):
```

Comment:
there's a PY3 variable that can be imported from common.py if it hasn't been imported already in this file